### ISSUE TITLE
Details about QueryTrack

### ISSUE THREAD

**Issue by DKWarcher** [hustvl/QueryInst#34]
Thank you for your nice work!

Since the training code of QueryTrack is not released, I hope you can share the following training details with me:

1. How is the instance embedding extracted from the reference frame? Is the process the same as that in the target frame? In the target frame, the corresponding embeddings of the ground-truth are extracted through the DynamicConv and HungarianAssigner. 
2. Besides the instance segmentation loss of the target frame, do you also calculate it for the reference frame like CrossVIS?
3.  During inference, the matching factor is changed. Therefore, I wonder if the association process is changed compared to MaskTrackRCNN?

Thanks again.

**Comment by Haelles** [comment]
> Q1: YES. We apply HungarianMatcher to get the corresponding queries, and then use ground-truth boxes to extract RoI features.
> Q2: NO. The target frames only take part in tracking head at training phase.
> Q3: NO.

In the Dynamic Instance Embedding part, do the key frame share queries($q_{t-1}^{*}$ in formula(4) ) with the reference frame? Or another HungarianMatcher is needed to get the reference frame's exclusive queries?

**Comment by vealocia** [comment]
> 

Yes, we inference querytrack on reference frame (```with torch.no_grad()```) and apply another HungarianMatcher to get the assignment between queries and ground-truths.

**Comment by Haelles** [comment]
> Yes, we inference querytrack on reference frame (`with torch.no_grad()`) and apply another HungarianMatcher to get the assignment between queries and ground-truths.
> 
> > > Q1: YES. We apply HungarianMatcher to get the corresponding queries, and then use ground-truth boxes to extract RoI features.
> > > Q2: NO. The target frames only take part in tracking head at training phase.
> > > Q3: NO.
> 
> > In the Dynamic Instance Embedding part, do the key frame share queries($q_{t-1}^{*}$ in formula(4) ) with the reference frame? Or another HungarianMatcher is needed to get the reference frame's exclusive queries?

Thank you! And could you please tell me the weights of `ClassificationCost`, `BBoxL1Cost` and `IoUCost` when getting the assignment between queries and ground-truth bboxes of the reference frame? Thanks again.
